### DISCUSSION TITLE
HELP: ADD BODY SLACK APPLICATION

### DISCUSSION THREAD

**Discussion by th0rinx** [nezhahq/nezhahq.github.io#70] [General]
Hi, I love the project and I am starting to implement it in my small business. 

I have a problem with sending notifications. 
I use SLACK-- I managed to synchronize the app with slack and it sends messages to my channel but does not send the data from my servers. 
Here is the body and configuration.

URL : xxxxxxx
request method: POST
request type: JSON
BODY: 
{
    "content": "#NEZHA#",
    "serverName": "#SERVER.NAME#",
    "ServerIP": "#SERVER.IP#",
    "ServerIPV4": "#SERVER.IPV4#",
    "ServerIPV6": "#SERVER.IPV6#",
    "CPU": "#SERVER.CPU#",
    "MEM": "#SERVER.MEM#",
    "SWAP": "#SERVER.SWAP#",
    "DISK": "#SERVER.DISK#",
    "NetInSpeed": "#SERVER.NETINSPEED#",
    "NetOutSpeed": "#SERVER.NETOUTSPEED#",
    "TransferIn": "#SERVER.TRANSFERIN#",
    "TranferOut": "#SERVER.TRANSFEROUT#",
    "Load1": "#SERVER.LOAD1#",
    "Load5": "#SERVER.LOAD5#",
    "Load15": "#SERVER.LOAD15#",
    "TCP_CONN_COUNT": "#SERVER.TCPCONNCOUNT", # Invalid.
    "UDP_CONN_COUNT": "#SERVER.UDPCONNCOUNT", # Invalid
}

**the error it tells me is. the error it tells me is. 
操作失败
请求错误：400@400 Bad Request missing_text_or_fallback_or_attachments
操作失败
请求错误：400@400 Bad Request missing_text_or_fallback_or_attachments**

someone help me to configure it ? thanks. 
Guillermo M

Translated with DeepL.com (free version)

**Comment by cantoblanco** [comment]
Hola Guillermo,

I do not use Slack as a notification channel, but I have tested it and found that Slack's webhook seems to **require a "text" key in the payload** when requesting the API.

Therefore, you can adjust the key name in the payload of your configuration and replace "content" with "text".

**Body:**
````
{
    "text": "#NEZHA#"
}
````
